sdk/keyvault/keyvault-certificates/CHANGELOG.md

```diff
@@ -1,11 +1,19 @@
 # Release History
 
-## 4.1.0-preview.2 (Unreleased)
+## 4.1.0 (Unreleased)
 
-- Using `Poller` and `PollerLike` from the latest `@azure/core-lro` instead of `KVPoller` and `KVPollerLike`.
+- Added the optional `serviceVersion` property to the `CertificateClient` optional parameters.
+  It defaults to the latest supported API version, which currently is `7.1`.
```

Comment:
```suggestion
- Added the optional `serviceVersion` property to the `CertificateClient` optional parameters.
      - It defaults to the latest service version, which currently is `7.1`.
      - Other supported service version at the moment is `7.0`.
      - If you were using `4.1.0-preview.1` version of this package, then this was previously named `apiVersion`.
```

Same feedback for the other 2 packages

Reply:
Thank you. I thought you didn't want the preview package to be mentioned, but I get it now! I like this.